templates/shared/forms/interactive/osm-contact-us.html

```diff
@@ -0,0 +1,254 @@
+<div class="p-modal" id="contact-modal">
```

Comment:
Currently this form is throwing a 403 error on submit. Could you try this fix https://github.com/canonical-web-and-design/ubuntu.com/pull/10097/files ? There was this issue with another contact form and this fixed it